packages/create-block/templates/javascript/edit.jsx.mustache

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
imo we should remove all of these comments as well. I find myself removing them when I'm editing blocks created by this package because they add a lot of noise to the file.

Reply:
I removed these comments.